jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -393,8 +477,8 @@ object XGBoost extends Serializable {
    * @param modelPath The path of the file representing the model
    * @return The loaded model
    */
-  def loadModelFromHadoopFile(modelPath: String)(implicit sparkContext: SparkContext):
-      XGBoostModel = {
+  def loadModelFromHadoopFile(modelPath: String)
+                             (implicit sparkContext: SparkContext): XGBoostModel = {
```

Comment:
why need to change to this style?

Reply:
we don't :) This style is more in line with what I've observed in other Scala projects, but certainly if this codebase uses a different convention then I can change it back. Just let me know.

Reply:
let's follow Spark's style, https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/rdd/RDD.scala#L398